.github/workflows/deploy.yaml

```diff
@@ -118,3 +118,21 @@ jobs:
                   echo "Deploying $IMAGE to $SERVICE_ID"
 
                   curl -sS --fail-with-body --request POST "https://api.render.com/v1/services/$SERVICE_ID/deploys" --header "authorization: Bearer ${{ secrets.RENDER_API_KEY }}" --data "{ \"imageUrl\": \"$IMAGE\" }"
+
+    deploy_connect_ui:
+        if: inputs.service == 'connect_ui'
```

Comment:
you need to add `connect_ui` to the list of `input.service` at the top of this file?

Reply:
🤦🏻